### PR TITLE
Hackit mail

### PR DESCRIPTION
**Pull request by Off3line**

### CODE REVIEW

File: mail-function/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/.next
+/.nyc_output
+/cypress-coverage
+/cypress/videos
```
Comment (BacLuc) [inline mail-function/.prettierignore:4]: unused

File: mail-function/package.json [hunk 2]
```diff
@@ -4,15 +4,30 @@
...
   "main": "index.js",
   "scripts": {
-    "exec": "src/index.js"
+    "exec": "src/index.js",
+    "eslint": "eslint '**/*.{js,jsx,ts,tsx}' && tsc",
+    "eslint-check": "eslint --no-fix '**/*.{js,jsx,ts,tsx}' && tsc",
+    "format": "prettier --write '**/*.{js,jsx,ts,tsx,css,html,md,json,scss}'",
+    "format-check": "prettier --check '**/*.{js,jsx,ts,tsx,css,html,md,json,scss}'",
+    "lint": "npm run eslint && npm run format",
+    "fixer": "eslint --fix --ext .js,.jsx .",
```
Comment (BacLuc) [inline mail-function/package.json:13]:
```suggestion
```
the --fix option is the default, thus its the same as the eslint above.

File: mail-function/package.json [hunk 3]
```diff
@@ -4,15 +4,30 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "exec": "src/index.js"
+    "exec": "src/index.js",
+    "eslint": "eslint '**/*.{js,jsx,ts,tsx}' && tsc",
```
Comment (BacLuc) [inline mail-function/package.json:8]:
```suggestion
    "eslint": "eslint '**/*.{js,jsx,ts,tsx}'",
```
There is no typescript, so you don't have a typescript compiler (tsc)

File: mail-function/package.json [hunk 4]
```diff
@@ -4,15 +4,30 @@
...
+    "dotenv-cli": "7.1.0",
+    "nodemailer": "^6.9.1"
+  },
+  "devDependencies": {
+    "eslint": "^8.38.0",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "lint": "^0.8.19",
```
Comment (BacLuc) [inline mail-function/package.json:30]:
```suggestion
```
What is the lint package?

File: mail-function/package.json [hunk 5]
```diff
@@ -4,15 +4,30 @@
...
-    "dotenv-cli": "7.1.0"
+    "dotenv-cli": "7.1.0",
+    "nodemailer": "^6.9.1"
+  },
+  "devDependencies": {
+    "eslint": "^8.38.0",
+    "eslint-config-standard": "^17.0.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^15.7.0",
+    "eslint-plugin-promise": "^6.1.1",
```
Comment (BacLuc) [inline mail-function/package.json:29]: you maybe also need eslint-plugin-prettier

File: mail-function/.prettierrc [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "semi": true,
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "printWidth": 90,
+  "jsonRecursiveSort": true,
```
Comment (BacLuc) [inline mail-function/.prettierrc:8]: for this you also need the package prettier-plugin-sort-json

File: mail-function/.prettierrc [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  "importOrderSeparation": true,
+  "importOrderSortSpecifiers": true
```
Comment (BacLuc) [inline mail-function/.prettierrc:10]: For this you need the plugin prettier-plugin-packagejson

File: mail-function/src/.eslintrc.json [hunk 8]
```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "browser": true,
```
Comment (BacLuc) [inline mail-function/src/.eslintrc.json:3]:
```suggestion
```
You run your application with nodejs and not in the browser, so you don't need the browser env

File: mail-function/src/order-mailer.js
Comment (BacLuc) [inline mail-function/src/order-mailer.js:1]: you maybe want to put this code into the index.js, and remove the code currently in the index.js

**Review by BacLuc — APPROVED** [review]
how did you manage to have a merge conflict with yourself?
I would rebase onto the main branch

File: mail-function/src/.eslintrc.json (hunk 8 above)
Comment (Off3line) [reply mail-function/src/.eslintrc.json:3]: yes thats weird, when configuting eslint I selected node .. but thanks

**Review by Off3line — COMMENTED** [review]
Updated according to your feedback.

**Pull request merged**